src/client/java/teammates/client/scripts/sql/VerifyCourseEntityAttributes.java

```diff
@@ -154,14 +163,180 @@ private boolean verifyStudent(CourseStudent oldStudent,
     }
 
     private List<Student> getNewStudents(String courseId) {
-        HibernateUtil.beginTransaction();
+        // HibernateUtil.beginTransaction();
```

Comment:
Is a transaction not needed to get the students here?

Reply:
We run into `IllegalStateException: Transaction already active`.